site/components/pages/Redesign/index.tsx

```diff
@@ -48,85 +48,175 @@ export const Home: NextPage<Props> = (props) => {
       <PageWrap>
         <HeaderDesktop
           buttons={[
-            <ButtonPrimary key="Enter App" label="Enter App" href={urls.app} />,
+            <ButtonPrimary
+              key="Enter App"
+              label={t({ message: "Enter App" })}
+              href={urls.app}
+            />,
           ]}
         >
-          <NavItemDesktop url={urls.home} name="Home" active={true} />
+          <NavItemDesktop
+            url={urls.home}
+            name={t({ message: "Home", id: "mainNav.home" })}
+            active={true}
+          />
           <NavItemDesktop
             url={urls.tutorial}
-            name="Buy Klima"
+            name={t({ message: "Buy Klima", id: "mainNav.buyKlima" })}
             target="_blank"
             rel="noreferrer noopener"
           />
-          <NavItemDesktop url={urls.stake} name="Stake" />
-          <NavItemDesktop url={urls.wrap} name="Wrap" />
-          <NavItemDesktop url={urls.bond} name="Bond" />
+          <NavItemDesktop
+            url={urls.stake}
+            name={t({ message: "Stake", id: "mainNav.stake" })}
```

Comment:
it should work without the ids too:
```
t`Stake`
```
it auto-generates an id based on the text. more fun for devs